criar.sql

```diff
@@ -34,37 +33,37 @@ CREATE TABLE Document (
 	date_fin	DATE,
 	type		TEXT,
 	ownerID		INTEGER,
-	CONSTRAINT document_type_fk FOREIGN KEY (type) REFERENCES Type(TID) ON DELETE RESTRICT,
-	CONSTRAINT document_owner_fk FOREIGN KEY (ownerID) REFERENCES Worker(WID) ON DELETE SET NULL,
-	CONSTRAINT checkDiDf CHECK(date_ini<date_fin OR date_fin=NULL)
+	CONSTRAINT document_type_fk FOREIGN KEY (type) REFERENCES Type(TID) ON DELETE RESTRICT ON UPDATE CASCADE,
+	CONSTRAINT document_owner_fk FOREIGN KEY (ownerID) REFERENCES Worker(WID) ON DELETE SET NULL ON UPDATE CASCADE,
+	CONSTRAINT checkDiDf CHECK (date_ini<date_fin OR date_fin=NULL)
 );
 CREATE TABLE Language (
 	LID			INTEGER,
-	Lname		TEXT NOT NULL,
+	Lname		TEXT UNIQUE NOT NULL,
 	CONSTRAINT language_pk PRIMARY KEY (LID)
 );
 CREATE TABLE File (
 	FID			INTEGER,
 	docID		INTEGER NOT NULL,
-	Fname		TEXT,
+	Fname		TEXT UNIQUE NOT NULL,
 	size		INTEGER,

```

Comment:
```suggestion
	size		INTEGER NOT NULL,
```
Size NULL também não me parece bem